api/v1/ptpconfig_webhook.go

```diff
@@ -190,6 +218,195 @@ func (r *PtpConfig) validate() error {
 				}
 			}
 		}
+
+		// validate secret-related settings for this profile
+		saFilePath, err := getSaFileFromPtp4lConf(conf)
+		if err != nil {
+			return fmt.Errorf("failed to get sa file path from ptp4lConf: %w", err)
+		}
+
+		// Skip security validation if sa_file is not configured (auth disabled)
+		if saFilePath == "" {
+			continue
+		}
+
+		if err := validateSaFile(saFilePath); err != nil {
+			return fmt.Errorf("failed to validate sa file: %w", err)
+		}
+
+		// Validate spp values per-interface (not global) exist in the secret
+		// [global] has spp -1 which doesn't need to exist in the secret
```

Comment:
indicate that spp -1 means that auth is disabled globally, not actually referencing an spp number